web_scraping/browser.py

```diff
@@ -0,0 +1,102 @@
+from pathlib import Path
+import concurrent.futures
+import requests
+import json
+from bs4 import BeautifulSoup
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from bs4 import BeautifulSoup
+import base64
+import asyncio
+import aiohttp
+
+chrome_options = Options()
+chrome_options.add_argument("--headless")
+
+directory = Path("output")
+
+async def get_html(url, session):
```

Comment:
We usually write the return type and the types of the variables for each function